po/common/en.po

```diff
@@ -6946,4 +6946,46 @@ msgstr "You cannot remove the main contact. Change it first."
 
 msgctxt "open_in_crm"
 msgid "Open in CRM"
-msgstr "Open in CRM"
\ No newline at end of file
+msgstr "Open in CRM"
+
+msgctxt "knowledge_panels_did_you_know"
+msgid "Did you know?"
+msgstr "Did you know?"
+
+msgctxt "knowledge_panels_what_you_can_do"
+msgid "What you can do"
+msgstr "What you can do"
+
+msgctxt "knowledge_panels_ingredients_rare_crops_title"
+msgid "Good for agricultural biodiversity"
+msgstr "Good for agricultural biodiversity"
+
+msgctxt "knowledge_panels_ingredients_rare_crops_subtitle"
+msgid "Contains neglected or underutilized crops:"
+msgstr "Contains neglected or underutilized crops:"
+
+msgctxt "knowledge_panels_ingredients_rare_crops_did_you_know"
+msgid "Since 1900, 75% of plant diversity has been lost as farmers around the world have abandoned local varieties for high-yielding varieties adapted to mass industrial processing. As a result, 60 % of human energy intake in the world comes from only 3 species: wheat, rice and corn."
+msgstr "Since 1900, 75% of plant diversity has been lost as farmers around the world have abandoned local varieties for high-yielding varieties adapted to mass industrial processing. As a result, 60 % of human energy intake in the world comes from only 3 species: wheat, rice and corn."
+
+msgctxt "knowledge_panels_ingredients_rare_crops_what_you_can_do"
+msgid "<p>Consuming diverse cereals, legumes, vegetables and fruits contributes to:</p>"
+"<ul>"
+"<li>preserving neglected and underutilized species</li>"
+"<li>rural development and support of local farmers and companies</li>"
+"<li>preserving of diversified landscapes</li>"
+"<li>food security</li>"
+"<li>and your health!</li>"
+"</ul>"
```

Comment:
could you break that down in strings that do not require HTML markup ?
Unrelated, We could use a list element in the KP syntax.

Reply:
We can, but there may not be enough context to have gramatically correct translations in all languages.